pkgs/applications/office/kmymoney/default.nix

```
@@ -10,24 +11,24 @@
 # Needed for running tests:
 , qtbase, xvfb-run
 
```

```suggestion

, python3
```
(needed for https://github.com/NixOS/nixpkgs/pull/183973/files#r935577346)